dbt/include/global_project/macros/materializations/snapshots/snapshot.sql

```diff
@@ -24,7 +23,7 @@
   {{ run_hooks(pre_hooks, inside_transaction=True) }}
 
   {% set strategy_macro = strategy_dispatch(strategy_name) %}
-  {% set strategy = strategy_macro(model, "snapshotted_data", "source_data", config, target_relation_exists) %}
+  {% set strategy = strategy_macro(model, "snapshotted_data", "source_data", model['config'], target_relation_exists) %}
```

Comment:
I commented on this in the `dbt-spark` PR as well; same question, what prompted the change?

Reply:
I wanted to use a "property" from the config object, and the dictionary form that was being used had no access to it.